README.md

```diff
@@ -68,21 +66,19 @@ requirement, since it is not used universally (e.g., it is not necessary on NERS
 
 
 ### Julia depot path
-There is no clear consensus where the Julia depot folder (by default on
-Unix-like systems: `~/.julia`) should be located.  On some systems that have
-good I/O connectivity, it resides in the user's home directory, e.g., at NERSC.
-On other systems, e.g., at CSCS, it is put on a scratch file system. At the time
+Since the available file systems can differ significantly between HPC centers, it is hard to make a general statement about where the Julia depot folder (by default on
+Unix-like systems: `~/.julia`) should be placed (via [`JULIA_DEPOT_PATH`](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH)). Generally speaking, the file system hosting the the Julia depot should have
+* good (parallel) I/O
+* no tight quotas
+* read and write access
+* shouldn't be automatically wiped on a regular basis (or should be excluded as an exception)
+
+On some systems, it resides in the user's home directory (e.g. at NERSC). On other systems, it is put on a parallel scratch file system (e.g. CSCS and PC2). At the time
 of writing (April 2022), there does not seem to be reliable performance data
 available that could help to make a data-based decision.
 
-If the depot path, which can be controlled by the
-[`JULIA_DEPOT_PATH`](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH)
-variable, is located on a scratch/workspace file system with automatic deletion
-of unused files, it must be ensured that there is a mechanism (either
-operator-provided or documented and in userspace) to prevent the deletion of
-files.
-In case multiple platforms share a single home directory, it might
-make sense to make the depot path platform dependend by setting the
+If multiple platforms, e.g. systems with different architecture, would access the same Julia depot, for example because the fiile system is shared, it might
```

Comment:
```suggestion
If multiple platforms, e.g., systems with different architecture, would access the same Julia depot, for example because the file system is shared, it might
```